cloudhsm/jdk-jce-image/Dockerfile

```
@@ -1,9 +1,8 @@
 # Base builder image to build apps that need to talk to the HSM cient
+FROM amazonlinux:2.0.20191016.0
 
-FROM amazonlinux:2.0.20190508
-
-# Install JDK
-RUN amazon-linux-extras install java-openjdk11
+# Install Java
+RUN yum install -y java-11-amazon-corretto
```

JAVA_HOME needs to be added as an environment variable to make it work.

```suggestion
RUN yum install -y java-11-amazon-corretto
ENV JAVA_HOME /usr/lib/jvm/java-11-amazon-corretto.x86_64
```